### PR TITLE
Authentication and user creation by administrators

### PR DESCRIPTION
**Pull request by briaguya**
Implemented basic user management (creation) as per issue #6

### CODE REVIEW

**Comment by briaguya** [comment]
I've updated the commit history to include clear messaging and documentation.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (jcbpl) [inline .gitignore]: Any objections to configuring your editor to add a newline at the end of files?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "atxfloodsadmin",
```
Comment (jcbpl) [inline package.json:2]: If we go with the super-repo approach do you think we should go ahead and move all these files into an `admin/` folder or whatever in the project root now?

File: src/AuthButton.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      Welcome! <button onClick={() => {
+        auth.signout(() => history.push('/'))
+      }}>Sign out</button>
+    </p>
+  ) : (
+    <p>You are not logged in.</p>
+  )
+));
+
+export default AuthButton;
```
Comment (jcbpl) [inline src/AuthButton.js]: Do you think we should think about a file structure convention at this point? Like putting this in `src/components/`.

**Comment by jcbpl** [comment]
Good commit messages! But don't be afraid to use "I". I like to think of them as if I were explaining what I did to a dev sitting next to me.

File: package.json (hunk 2 above)
Comment (briaguya) [inline package.json]: I wouldn't be opposed to it, but I'm also fine with waiting to move it until we start work on the public frontend.

File: src/AuthButton.js (hunk 3 above)
Comment (briaguya) [inline src/AuthButton.js]: screen shot 2017-06-02 at 12 16 27 pm

It hasn't gotten to a point where I'm having trouble managing it yet. I figure once we add some more stuff it'll become a more pressing issue.

**Pull request merged**